platform/viewer/src/appExtensions/GenericRoutesModule/index.js

```diff
@@ -0,0 +1,8 @@
+import routesModule from './routesModule.js';
+
+export default {
+  id: 'generic-routes-module',
+  getRoutesModule() {
+    return routesModule;
+  },
+};
```

Comment:
We have separate `index.js` extension files for `MeasurementsPanel` and `GenericViewerCommands` because we expect to move the `MeasurementsPanel` over entirely to the `<repo-root>/extensions/*` folder after its completely decoupled.

You can probably expose this module in the `genericViewerCommands/index.js`, or move the file up a level and expose bother from there.

Reply:
I didnt get your point here.. :(

Reply:
I guess I didnt get your point